API/src/main/java/com/bgsoftware/superiorskyblock/api/config/SettingsManager.java

```diff
@@ -236,12 +236,6 @@ public interface SettingsManager {
      */
     String getStackedBlocksProvider();
 
-    /**
-     * Whether inventory of island members should be cleared when their island is disbanded or not.
-     * Config-path: disband-inventory-clear
-     */
-    boolean isDisbandInventoryClear();
-
```

Comment:
Do not delete the method.
Instead mark it as deprecated, change the comment to reflect what it really returns (for example, return true if one of the options are true) and in the future it will be deleted